sass/_uswds-theme-custom-styles.scss

```diff
@@ -105,4 +105,8 @@ i.e.
 .usa-collection__body h3{
    margin-top: 0.6rem;
    margin-bottom: 0;
+}
+
+.usa-prose .usa-collection__item{
+   border-color: #DCDEE0; 
```

Comment:
I am curious where this color comes from. I don't think we are using it anywhere else. There is a [theme color](https://designsystem.digital.gov/design-tokens/color/theme-tokens/): `$theme-color-base-lighter` which is `#dfe1e2` that we use in other places (including the top/bottom borders on the side nav). Should this be the same color?

Reply:
The color is coming from the figma mock up, Kristen set it that color code, let me check with Kirsten if she is ok with using the base-lighter color.

Reply:
Confirmed with Kristen, she is fine with color code change, she copy it from USWDS theme color and thought it is correct. We both didn't catch the color code difference! Thanks